src/ape/api/providers.py

```diff
@@ -189,7 +189,7 @@ class ReceiptAPI:
     to not wait for confirmations.
 
     Get a receipt by making transactions in ``ape``, such as interacting with
-    a :class:`ape.contracts.base.ContractInstance`.
+    a ``ContractInstance``.
```

Comment:
I don't think change makes sense to do

Reply:
Find and replace error, thanks for catching it!